### PR TITLE
Request to Merge readme game instructions to development branch

### PR DESCRIPTION
**Pull request by sun1lkumar**
Game instructions have been given to users who want to play the tictactoe.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,2 +1,88 @@
...
+To run this project, you need a computer with Ruby installed.
+> If you are uncertain on how to install Ruby, check out this [tutorial](https://www.theodinproject.com/courses/ruby-programming/lessons/installing-ruby-ruby-programming) made by [The Odin Project](https://www.theodinproject.com/about).
+
+### How to Play
+#### In your local enviroment:
+
+1. Clone the repository:
+```
+https://github.com/VanessaAoki/TicTacToe.git
+```
```
Comment (GzimAsani) [inline README.md:22]: Please tell your viewers that they have to first **cd** inside the game

**Review by GzimAsani — CHANGES_REQUESTED** [review]
###  Status: Changes Required :recycle:
**Hello team** very nice job reaching this far, your game looks really god and your README looks almost completed, for requirements please check comments below:
- [OPTIONAL] Your readme will look more professional if you could add picture examples of winning and a draw case in the how to play section about.

### **Please make the required changes and submit another review request when you are ready.**

<hr>
### if you have any question kindly reach me on slack @GzimAsani

### HAPPY CODING :computer:

File: README.md (hunk 1 above)
Comment (SunilDevalla) [reply README.md:22]: Thank you for the speedy feedback on this, we will resolve the issues and request another review.

**Review by GzimAsani — APPROVED** [review]
# **STATUS: APPROVED** 
Congratulations **TEAM**  , Great job implementing the requirements, this project is approved and you can move to your next project. Great project
Don't forget to submit a project completion form.
 Cheers! 👍🏽

<hr>
### if you have any question kindly reach me on slack @GzimAsani

### HAPPY CODING :computer:

**Pull request merged**